.editorconfig

```diff
@@ -6,19 +6,20 @@ root = true
 
 # Default settings:
 # A newline ending every file
-# Use 2 spaces as indentation
-# Trim trailing whitespace
+# Use 4 spaces as indentation
 [*]
 insert_final_newline = true
```

Comment:
{F702, layout=inline, width=20} This line should be removed IMO. It behaves "poorly" when used with certain designers and the project system and doesn't offer much.